src/eventHandlers/BulkMessageDeleteHandle.js

```diff
@@ -16,7 +14,7 @@ export default class LogMessageDelete extends Handler {
    * @param {Genesis} bot  The bot object
    */
   constructor(bot) {
-    super(bot, 'handlers.logMessageDeleteBulk', Events.MESSAGE_DELETE_BULK);
+    super(bot, 'handlers.logMessageDeleteBulk', Events.MessageDelete);
```

Comment:
i think this is the wrong event. this would duplicate the same events as MessageDeleteHandle